tools/analysis/maian/docker-compose.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+  maian:
+    image: cryptomental/maian-augur-ci
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    entrypoint: ["sleep", "infinity"]
```

Comment:
Given you gave instructions in the readme on how to run it, should we include this compose?